torch/nested/__init__.py

```diff
@@ -41,7 +41,7 @@ def nested_tensor(tensor_list: List[Tensor], *, dtype: Optional[DType] = None, d
         >>> nt.is_leaf
         True
     """
-    if not isinstance(tensor_list, list) or any([not torch.is_tensor(t) for t in tensor_list]):
+    if not isinstance(tensor_list, list) or any([not isinstance(t, torch.Tensor) for t in tensor_list]):
```

Comment:
Why this change?

Reply:
I think it was done automatically from a linter suggestion, I can revert it